features/energy.py

```diff
@@ -28,7 +28,12 @@ def EnergyJob(crp, energy_options=None, **kwargs):
     )
 
 
-def energy_flow(without_samples=False, samples_options={}, fft_options={}):
+def energy_flow(
+    without_samples=False,
+    normalization_type="divide-by-mean",
```

Comment:
It might be better to add new parameters last, in case people did not call this function with named arguments, but for me this is ok.